features/PSE001_Ifcpropertyset-name.feature

```diff
@@ -0,0 +1,27 @@
+@implementer-agreement
+@PSE
+Feature: PSE001 - IfcPropertySet Name
+The rule verifies that each IfcPropertySet Name attribute is defined correctly.
```

Comment:
The feature starts with 'The rule verifies that ..'. The features name matches the name of feature_name.feature filename, all the testfiles are there including a readme and they are correctly named. 
Nice :)